docs/overview/office-add-in-availability.md

```diff
@@ -17,6 +17,9 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 > [!NOTE]
 > The initial Office 2016 release installed via MSI only contains the ExcelApi 1.1, WordApi 1.1, and Common API requirement sets. For more information about the update history of the various Office versions, check out the [See also](#see-also) section.
 
+> [!NOTE]
```

Comment:
Should we have two consecutive notes? Maybe consider merging into one note (2 parts, of course) or changing from NOTE to IMPORTANT.

Reply:
I'd combine to one note with 2 paragraphs since both have to do with versions.